common/pinot/response_utility.go

```diff
@@ -83,12 +83,23 @@ func ConvertSearchResultToVisibilityRecord(hit []interface{}, columnNames []stri
 	if err != nil {
 		logger.Error("Unable to Unmarshal searchAttribute map", tag.Error(err))
 	}
+	var memo *p.DataBlob
+	if attributeMap["Memo"] != nil {
+		err = json.Unmarshal([]byte(fmt.Sprintf("%s", attributeMap["Memo"])), &memo)
```

Comment:
no need for sprintf, just 

```suggestion
		err = json.Unmarshal([]byte(attributeMap[MemoKey]), &memo)
```